src/test/resources/alma/HT005207972.json

```diff
@@ -115,7 +115,7 @@
     },
     "resultOf" : {
       "type" : [ "CreateAction" ],
-      "endTime" : "2021-02-15T08:30:45",
+      "endTime" : "2021-02-15T12:54:38",
```

Comment:
Omit the adding of this file as discussed.

Reply:
did that.

Reply:
+1